### PR TITLE
Mapping tweaks to match Controllers spec

### PR DESCRIPTION
**Pull request by ZappoMan**
* hydra 3,4 buttons to primary thumb
* hydra 1,2 buttons to secondary thumb
* standard left joystick X now rotates yaw
* standard right joystick X now strafes
* also adds the "oculus touch and gestures" to the Standard mapping for access by scripts

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: d359aa3b654a6235da3e651bdfb05b5383be2982 available at: https://gist.github.com/14c254203bdb83c62b17<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6275.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6275.dmg

File: libraries/controllers/src/controllers/StandardController.cpp [hunk 1]
```diff
@@ -55,6 +55,9 @@ Input::NamedVector StandardController::getAvailableInputs() const {
         makePair(LS, "LS"),
         makePair(RS, "RS"),
 
+        makePair(LS, "LS_TOUCH"),
```
Comment (jherico) [inline libraries/controllers/src/controllers/StandardController.cpp]: the canonical forms of the names should be camelCase.  Also these should be using new channel identifiers, 

`makePair(LS_TOUCH, "LSTouch"),`

File: libraries/controllers/src/controllers/StandardControls.h [hunk 2]
```diff
@@ -39,16 +39,33 @@ namespace controller {
         // These don't map to SDL types
         LEFT_PRIMARY_THUMB,
         LEFT_SECONDARY_THUMB,
+        LEFT_PRIMARY_THUMB_TOUCH,
```
Comment (jherico) [inline libraries/controllers/src/controllers/StandardControls.h]: if we want to expose everything from Oculus Touch, we need A/B/X/Y_TOUCH, as well as LT/RT_TOUCH

**Comment by hifi-gustavo** [comment]
Build console output for commit: f532607ccab1b94c150c6f207dbf49557c0148ad available at: https://gist.github.com/14c254203bdb83c62b17<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6275.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6275.dmg

File: libraries/controllers/src/controllers/StandardController.cpp (hunk 1 above)
Comment (ZappoMan) [inline libraries/controllers/src/controllers/StandardController.cpp]: typo - my bad - will fix

File: libraries/controllers/src/controllers/StandardControls.h (hunk 2 above)
Comment (ZappoMan) [inline libraries/controllers/src/controllers/StandardControls.h]: I thought LeftPrimaryThumbTouch was sufficient

**Comment by hifi-gustavo** [comment]
Build console output for commit: 02288e84e8017943f2507d4be87dc11985f39712 available at: https://gist.github.com/14c254203bdb83c62b17<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6275.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6275.dmg

**Pull request merged**